test/framework/src/main/java/org/opensearch/bootstrap/BootstrapForTesting.java

```diff
@@ -280,6 +284,25 @@ public boolean implies(ProtectionDomain domain, Permission permission) {
         }
         return raw;
     }
+    
+    /**
+     * Collect host addresses of all local interfaces so we could check 
+     * if the network connection is being made only on those.
+     * @return host names and addresses of all local interfaces
+     */
+    private static Set<String> getTrustedHosts() {
```

Comment:
Had to extend the list of trusted host to all local interfaces, some tests used non-loopback addresses